base/.eslintrc.json

```diff
@@ -1,21 +1,45 @@
 {
   "extends": [
+    "eslint-config-prettier",
+    "eslint:recommended",
+    "plugin:import/recommended",
     "plugin:prettier/recommended",
-    "plugin:import/recommended"
+    "prettier",
+    "prettier/standard"
   ],
+  "plugins": ["prettier"],
   "settings": {
     "import/resolver": "webpack"
   },
   "rules": {
+    "prettier/prettier": 2,
```

Comment:
This is coming from the documentation but I am not sure if we should have this? @JakeSidSmith

Reply:
Keeping it, as it is the line activates prettier type checking.